### PR TITLE
chore: remove chaincode on codebase

### PR DESCRIPTION
**Pull request by ThibaultFy**
## Description

close FL-1349

## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Review by oleobal — APPROVED** [review]
Looks fine to me, but others are better at backending

**Comment by ThibaultFy** [comment]
> Looks fine to me, but others are better at backending

Maybe @guilhem-barthes you can have a look ? 🙏

File: examples/values/backend-org-1.yaml [hunk 1]
```diff
@@ -125,10 +125,6 @@ orchestrator:
...
         restricted: false
         model_export_enabled: true
-        chaincode:
-          name: mycc
     - yourchannel:
         restricted: false
-        model_export_enabled: true
-        chaincode:
-          name: yourcc
+        model_export_enabled: true
```
Comment (guilhem-barthes) [inline examples/values/backend-org-1.yaml:130]: Add a newline here plz

**Review by guilhem-barthes — APPROVED** [review]
LGTM

**Pull request merged**